app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] The view for the Login page should be modified to match the wireframes provided, kindly consider implementing this so your web app looks like the wireframes. you can style the app however you like but it should follow the wireframe (you can center the input fields and give them some padding, also there is no label kindly use placeholders instead. The button should be centered and take the whole width to match the input fields) 

|Your app|Wireframe|
|-|-|
|![Screenshot_20230727_112715](https://github.com/christianonoh/beyond-borders/assets/25227022/3b904013-57e4-4da8-937d-77b502218614)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)|